packages/gatsby-plugin-page-creator/src/is-valid-collection-path-implementation.ts

```diff
@@ -1,12 +1,15 @@
 import sysPath from "path"
-import reporter from "gatsby-cli/lib/reporter"
+import { Reporter } from "gatsby"
```

Comment:
Should this be a type only import? 

```suggestion
import type { Reporter } from "gatsby"
```